recipes/libignition-rendering/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set base_name = "libignition-rendering" %}
+{% set version = "4.1.0" %}
+{% set name = base_name  + version.split('.')[0] %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ignitionrobotics/ign-rendering/archive/ignition-rendering{{ version.split('.')[0] }}_{{ version }}.tar.gz
+  sha256: 465e858332d5b0e608888e9d62d08f779c7d99d0a51f2de7ba23ba640cdd4be4
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+    - make                               # [not win]
+    - pkg-config                         # [not win]
+
+    # x11
+    - {{ cdt('mesa-libgl-devel') }}      # [unix]
+    - {{ cdt('mesa-dri-drivers') }}      # [unix]
+    - {{ cdt('libselinux') }}            # [linux]
+    - {{ cdt('libxdamage') }}            # [linux]
+    - {{ cdt('libxxf86vm') }}            # [linux]
+    - {{ cdt('libxfixes') }}             # [linux]
+    - {{ cdt('libxext') }}               # [linux]
+    - {{ cdt('libxau') }}                # [linux]
+  host:
+    - xorg-libx11                        # [unix]
+    - xorg-libxext                       # [unix]
+    - libglu                             # [linux]
+    - libignition-math6 >=6.6
+    - libignition-common3
+    - libignition-plugin1
+    - ogre
+  run:
+    - xorg-libx11                        # [unix]
+    - xorg-libxext                       # [unix]
+    - libglu                             # [linux]
+    - libignition-math6 >=6.6
+    - libignition-common3
+    - libignition-plugin1
+    - ogre
+    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx and x86_64]
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/ignition/rendering{{ version.split('.')[0] }}/ignition/rendering.hh  # [not win]
+    - test -f ${PREFIX}/lib/libignition-rendering{{ version.split('.')[0] }}.so  # [linux]
+    - test -f ${PREFIX}/lib/libignition-rendering{{ version.split('.')[0] }}.dylib  # [osx]
+    - test -f ${PREFIX}/lib/cmake/ignition-rendering{{ version.split('.')[0] }}/ignition-rendering{{ version.split('.')[0] }}-config.cmake  # [not win]
+    - if exist %PREFIX%\\Library\\include\\ignition\\rendering{{ version.split('.')[0] }}\\ignition\\rendering.hh (exit 0) else (exit 1)  # [win]
+    - if exist $PREFIX$\\Library\\lib\\ignition-rendering{{ version.split('.')[0] }}.lib (exit 0) else (exit 1)  # [win]
+    - if exist $PREFIX$\\Library\\bin\\ignition-rendering{{ version.split('.')[0] }}.dll (exit 0) else (exit 1)  # [win]
+    - if exist %PREFIX%\\Library\\lib\\cmake\\ignition-rendering{{ version.split('.')[0] }}\\ignition-rendering{{ version.split('.')[0] }}-config.cmake (exit 0) else (exit 1)  # [win]
+
+about:
+  home: https://github.com/ignitionrobotics/ign-rendering
+  license: Apache-2.0
+  license_file: LICENSE
+  summary: Ignition Rendering
+
+extra:
+  recipe-maintainers:
+    - JShep1
```

Comment:
It's always good to have multiple maintainers.
Maybe @traversaro is interested? You can also add me (@wolfv).

Reply:
Ok for me.

Reply:
Sounds good! 5df9a9f8c76ad4180ae16bb13871f451eab094a2